src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -280,31 +280,9 @@ impl MetaData for S3MetaData {
     }
 
     #[instrument(skip(self))]
-    async fn forget(&self, ino: u64, nlookup: u64) -> DatenLordResult<bool> {
-        let (res, retry) = retry_txn!(TXN_RETRY_LIMIT, {
-            let mut txn = self.kv_engine.new_meta_txn().await;
-            let mut result = false;
-            let inode = self.get_inode_from_txn(txn.as_mut(), ino).await?;
-            inode.dec_lookup_count_by(nlookup);
-            let is_deleted = inode.get_lookup_count() == 0;
-            if is_deleted {
-                // FIXME: rename should also rename the node's name and reset the parent ino
-                txn.delete(&KeyType::DirEntryKey((
-                    inode.get_parent_ino(),
-                    inode.get_name().to_owned(),
-                )));
-                txn.delete(&KeyType::INum2Node(ino));
-                result = true;
-            } else {
-                txn.set(
-                    &KeyType::INum2Node(ino),
-                    &ValueType::Node(inode.to_serial_node()),
-                );
-            }
-            (txn.commit().await, result)
-        });
-        FILESYSTEM_METRICS.observe_storage_operation_throughput(retry, "forget");
-        res
+    async fn forget(&self, _ino: u64, _nlookup: u64) -> DatenLordResult<bool> {
+        FILESYSTEM_METRICS.observe_storage_operation_throughput(0_i32, "forget");
+        Ok(true)
```

Comment:
Return `false` here, otherwise this will always remove the file.

Reply:
OK, That make sense.